### PR TITLE
Lionelb/hasura user api pipe

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

File: hasura/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM hasura/graphql-engine:v1.2.1.cli-migrations-v2
+ENV HASURA_GRAPHQL_ENABLE_TELEMETRY false
+COPY ./migrations /hasura-migrations
```
Comment (lionelB) [inline hasura/Dockerfile:3]:
```suggestion
COPY ./migrations /hasura-migrations
COPY ./metadata /hasura-metadata
```

**Comment by revolunet** [comment]
LGTM !

File: hasura/migrations/1588758007277_init/up.sql [hunk 2]
```diff
@@ -1,4 +1,3 @@
-CREATE EXTENSION citext;
```
Comment (lionelB) [inline hasura/migrations/1588758007277_init/up.sql:1]: you need to remove it from the down.sql

Comment (lionelB) [reply hasura/migrations/1588758007277_init/up.sql:1]: but the create extension is needed to work on vanilla postgres

File: hasura/Dockerfile (hunk 1 above)
Comment (lionelB) [reply hasura/Dockerfile:3]: I don't understand why metadata are not needed ?

Comment (revolunet) [reply hasura/Dockerfile:3]: its been commited

File: hasura/migrations/1588758007277_init/up.sql (hunk 2 above)
Comment (revolunet) [reply hasura/migrations/1588758007277_init/up.sql:1]: mmm ok. the issue is that atm on azure PG only superuser can create extensions, that's why we add it on database creation

Comment (revolunet) [reply hasura/migrations/1588758007277_init/up.sql:1]: i try some tricks

**Comment by lionelB** [comment]
thanks julien !

**Pull request merged**